touchforms/formplayer/api.py

```diff
@@ -287,26 +290,36 @@ def get_response(data, url, auth=None):
     except Exception, e:
         raise e
 
-def sync_db(username, auth=None):
+
+def sync_db(username, domain=None, auth=None):
     data = {
         "action":"sync-db",
         "username": username
     }
-    response = post_data(json.dumps(data), settings.XFORMS_PLAYER_URL, "text/json", auth)
+
+    if domain and USE_FORMPLAYER.enabled(domain):
+        response = post_data(json.dumps(data), settings.FORMPLAYER_URL + "/sync-db", "application/json", auth)
+    else:
+        response = post_data(json.dumps(data), settings.XFORMS_PLAYER_URL, "text/json", auth)
     response = json.loads(response)
     return response
 
 
-def get_raw_instance(session_id, auth=None):
+def get_raw_instance(session_id, domain=None, auth=None):
     """
     Gets the raw xml instance of the current session regardless of the state that we're in (used for logging partially complete
     forms to couch when errors happen).
     """
+
     data = {
         "action":"get-instance",
         "session-id": session_id,
         }
-    response = post_data(json.dumps(data), settings.XFORMS_PLAYER_URL, "text/json", auth)
+
+    if domain and USE_FORMPLAYER.enabled(domain):
+        response = post_data(json.dumps(data), settings.FORMPLAYER_URL + "/get-instance", "application/json", auth)
```

Comment:
it may be best to stop passing the URL into `post_data` and `get_response` and just pass the domain in and then figure out the URL in post_data based on the domain. that was we have less switch statements